xml/System.Runtime.InteropServices/SuppressGCTransitionAttribute.xml

```diff
@@ -51,6 +51,8 @@ This attribute is ignored if applied to a method without the <xref:System.Runtim
  * Immediate runtime termination.
  * Data corruption.
 
+ Applying this attribute impacts when the export binding of the P/Invoke is performed. This can result in <xref:System.EntryPointNotFoundException> or other exceptions being thrown earlier than when the attribute is not applied.
```

Comment:
```suggestion
 Applying this attribute can cause the P/Invoke export to be bound earlier as side-effect of the calling method being JIT compiled. This can result in <xref:System.EntryPointNotFoundException> or other exceptions being thrown earlier than when the attribute is not applied. 
```
We can be more specific about the timing differences.